examples/jobs/passjob_custom.py

```diff
@@ -6,17 +6,15 @@
 from avocado.core.nrunner import Runnable
 from avocado.core.suite import TestSuite
 
-config = {'run.test_runner': 'nrunner'}
-
 # Custom method (no discovery, no guess, no magic)
 # Since there is no magic, we need to pass a suite name, otherwise a uuid4 will
 # be used for suite.name. Also resolver.references will be ignored (Avocado will not
 # creating tests suites for you).
 
-suite1 = TestSuite(config=config,
+suite1 = TestSuite(config={},
                    tests=[Runnable("noop", "noop")], name='suite1')
-suite2 = TestSuite(config=config,
+suite2 = TestSuite(config={},
```

Comment:
There is any special need to pass an empty config here? IIRC, config is optional for TestSuite and for Job.

Reply:
I was on the fence about hinting that users can pass a config (as this is an example) or not... I'm still on the fence :)

Reply:
I guess looking at the other examples, I'm not on the fence anymore.  I'll remove them.